pkg/source/util/util.go

```diff
@@ -0,0 +1,81 @@
+package util
```

Comment:
Can we move this to `pkg/source` instead? I don't believe that we need a separate util package when the code aligns pretty closely with `pkg/source/unpacker.go`

Reply:
@tylerslaton   Thanks for review. The util/util.go imports internal/source and go files in internal/source import pkg/source so the util/util.go cannot be in the pkg/source.  It causes circular dependencies. So it is put in a separate package.